modules/Campaigns/WizardMarketing.html

```diff
@@ -838,7 +838,7 @@ <h4 class="header-4">{$MOD.LBL_MRKT_NAME}</h4>
 					</script>
 				</td>
 
-				<td>
+				<td class="date-alignment-marketing">
```

Comment:
Hi @serhiisamko091184, there are 2 approaches to class names:

- a name that represents the structured that is being represented, like `record`, `row`, `panel`
- a class that is generic and adds / changes css, like `mt-1` (mt for margin-top), `align-left`

And then the corresponding css is added for it. We should try one of the above or a mix.